src/app/ApplicationTemplate.UWP/Views/Content/Settings/EditProfilePage.xaml

```diff
@@ -30,8 +30,7 @@
 							 x:Uid="Profile_FirstName"
 							 Text="{Binding Form.FirstName, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
 							 ios:ReturnKeyType="Next"
-							 android:ImeOptions="Next"
-							 FontSize="16" />
+							 android:ImeOptions="Next" />
```

Comment:
shouldnt there be a nv:TextBoxBehavior.NextControl="{Binding ElementName=EmailBox}" where nv is nventive.view and ElementName the x:name of the next control you want to focus?